pkg/volume/csi/csi_plugin.go

```diff
@@ -72,6 +72,11 @@ type csiPlugin struct {
 	csiDriverInformer csiinformer.CSIDriverInformer
 }
 
+type inlineLifecycle string
+
+var persistentInlineLifecycle inlineLifecycle = "persistent"
```

Comment:
Add a TODO, we should define this in the API

Reply:
OK.